### ISSUE TITLE
aws_vpc: enable_dns_support not set correctly at initial creation

### ISSUE THREAD

**Issue by mattmoyer** [hashicorp/terraform#10168]
### Terraform Version
v0.7.11

### Affected Resource(s)
- aws_vpc

### Terraform Configuration Files
```hcl
resource "aws_vpc" "test" {
  cidr_block           = "10.0.0.0/16"
  enable_dns_support   = false
}

output "vpc_id" {
    value = "${aws_vpc.test.id}"
}
```

### Debug Output
https://gist.github.com/mattmoyer/38cc4322ed6a4ee5dc128e34935d61a8

### Expected Behavior
The VPC should be created with `enableDnsSupport` set to `false` during initial creation.

### Actual Behavior
The VPC is initially created with`enableDnsSupport` set to `true`, which is fixed by re-running `terraform apply`.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply` (initial apply using the `.tf` linked above)
2. `aws ec2 describe-vpc-attribute --vpc-id $(terraform output vpc_id) --attribute enableDnsSupport` (shows that DNS support was enabled).
3. `terraform apply` (second apply, notices the discrepancy and fixes it)
4. `aws ec2 describe-vpc-attribute --vpc-id $(terraform output vpc_id) --attribute enableDnsSupport` (shows that DNS support was correctly disabled).

I haven't really looked at the resource yet but I can probably take a shot at fixing this. Would you want it cherry picked into 0.7.x?

This could potentially be considered a security vulnerability since one of the reasons to disable DNS support is to lock down egress from a VPC.

**Comment by catsby** [comment]
Fixed in #10171 , thanks!

**Comment by mattmoyer** [comment]
Thanks @Ninir!
